retrace/meta.yaml

```diff
@@ -0,0 +1,18 @@
+{% set name = "retrace" %}
+{% set version = "android_10.0.0_r29" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # no checksum -- server returns different tarball every time...
+  url: https://android.googlesource.com/platform/prebuilts/r8/+archive/3769a499a855cfc4d0d1039eb2a0ea72ecdf7070.tar.gz
```

Comment:
FWIW, I also tried using `git_url: ...` + `git_rev: ...` but the repo is gigantic (couple gigs).
So I tried adding `git_depth: 1`, but that doesn't work because it first shallowly clones the default branch and then tries to checkout the rev.... :/
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-from-git